src/main/java/uk/gov/hmcts/darts/audio/component/AudioBeingProcessedFromArchiveQuery.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.darts.audio.component;
+
+import uk.gov.hmcts.darts.audio.model.AudioRequestBeingProcessedFromArchiveQueryResult;
+
+import java.util.List;
+
+public interface AudioBeingProcessedFromArchiveQuery {
+
+    List<AudioRequestBeingProcessedFromArchiveQueryResult> getResults(Integer hearingId);
```

Comment:
I'd find it a bit less confusing tbh if this result object was something different to match your query component - so `AudioBeingProcessedFromArchiveQueryResult`.  You might even be able to map directly to the generated `AudioMetadata`, so your query could do the channel1 filter and set the result fields in one hit rather than separately enriching the found entities in the `AudioController`?

Reply:
created new class for result - AudioBeingProcessedFromArchiveQueryResult.
left the rest of it alone though because a) it works ok b) individual methods could potentially reused for other purposes so combining would prevent this c) refactoring could have undesired affect of making code harder to understand